### PR TITLE
Week 3 Quiz Submission from sbagdi-stat6250

### PR DESCRIPTION
**Pull request by SurbhiBagdi**

### CODE REVIEW

File: week-03/README.md [hunk 1]
```diff
@@ -16,69 +16,95 @@ The instructor will then review the pull request and make comments should furthe
...
+- *Question* (sbagdi-stat6250): What does the DESCENDING option does? And to which statement should it be added?
+a) The DESCENDING option sorts observations in descending order of first variable (the one programmer specifies in VAR statement) within descending order of the second variable. And it should be added to the BY statement.
+b) The DESCENDING option sorts observations in ascending order of second variable (the one programmer specifies in VAR statement) within descending order of the first variable. And it should be added to the BY statement.
+c) The DESCENDING option sorts observations in ascending order of first variable (the one programmer specifies in VAR statement) within descending order of the second variable. And it should be added to the PROC PRINT statement.
+d) The DESCENDING option sorts observations in ascending order of first variable (the one programmer specifies in VAR statement) within descending order of the second variable. And it should be added to the PROC SORT statement.
 
 
 
 [Course Textbook Chapter 4, Problem 9]
+- *Question* (sbagdi-stat6250): Which operator is used to select observations that include a specified string?
```
Comment (prof-lankham) [inline week-03/README.md]: Invalid question for a Weekly Quiz; you're not creating a quiz for someone else. The instructions specify that you should be asking questions about SAS programming that **you** actually have, as prompted by concepts in textbook problems and recipes.

File: week-03/README.md [hunk 2]
```diff
@@ -16,69 +16,95 @@ The instructor will then review the pull request and make comments should furthe
...
 
 
 
 [Course Textbook Chapter 4, Problem 4]
-
+- *Question* (sbagdi-stat6250): What happens if a programmer doesn’t use OUT= option in a PROC SORT step? 
 
 
 [Course Textbook Chapter 4, Problem 7]
+- *Question* (sbagdi-stat6250): What does the DESCENDING option does? And to which statement should it be added?
```
Comment (prof-lankham) [inline week-03/README.md]: Invalid question for a Weekly Quiz; you're not creating a quiz for someone else. The instructions specify that you should be asking questions about SAS programming that **you** actually have, as prompted by concepts in textbook problems and recipes.

File: week-03/README.md [hunk 3]
```diff
@@ -16,69 +16,95 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Textbook Chapter 3, Problem 1]
+- *Question* (sbagdi-stat6250): Which of the following is true for SAS statements?
```
Comment (prof-lankham) [inline week-03/README.md]: Invalid question for a Weekly Quiz; you're not creating a quiz for someone else. The instructions specify that you should be asking questions about SAS programming that **you** actually have, as prompted by concepts in textbook problems and recipes.

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Please resolve feedback and DM me in Slack with the PR URL for re-evaluation

**Review by SurbhiBagdi — COMMENTED** [review]
Professor. In the files changes tab, I see the updated file. I hope it's what I had to do.

**Pull request merged**